Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -124,5 +127,10 @@
   <ItemGroup>
     <None Remove="mdoc.targets" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Contacts\Contacts.netstandard.tvos.watchos.cs">
+      <SubType></SubType>
+    </Compile>
+  </ItemGroup>
```

Comment:
VS borked the csproj :(

Reply:
Usually when working with files with multi-targeting, I do it in VSCode / explorer. VS can't figure things out correctly with all the conditions.

Reply:
Check out the diff and then just revert the csproj.

Reply:
I delete it and it reappears

Reply:
Make sure to close VS. There are 2 changes to revert in there.